lib/breadcrumbs.ts

```diff
@@ -126,8 +158,14 @@ function buildSearchResultBreadcrumbs(data) {
   return [];
 }
 
-function buildSchemaBreadcrumbs(schema, profile) {
-  if (!profile) {
+/**
+ * Generate the breadcrumb data for a schema page.
+ * @param schema {DatabaseObject | null} The schema object, or null for the schema index page
+ * @param title {string} The data type associated with a schema, e.g. "Biomarker"
+ * @returns {Breadcrumb[]} Breadcrumb data for the given schema
+ */
+function buildSchemaBreadcrumbs(schema: Schema, title: string): Breadcrumb[] {
```

Comment:
if the schema can be null, should be `schema?: Schema`

Reply:
Oh! That’s a comment mistake. `buildBreadcrumbs` only gets called when `data` is known to be a good object.